Simplenote/src/main/java/com/automattic/simplenote/utils/SystemBarUtils.java

```diff
@@ -0,0 +1,318 @@
+package com.automattic.simplenote.utils;
+
+import android.app.Activity;
+import android.content.res.Configuration;
+import android.os.Build;
+import android.view.View;
+import android.view.ViewGroup;
+import android.view.Window;
+import android.view.WindowInsetsController;
```

Comment:
Minor (🔍): Consider removing this unused import.

Reply:
removed 👍